src/test/java/org/ssu/edu/teachua/ui/TUA231.java

```diff
@@ -50,7 +50,7 @@ public void testTUA231() throws IOException {
                 .fillSortNumber(VALID_SORT_NUMBER)
                 .fillTitle(VALID_TITLE)
                 .fillDescription(VALID_DESCRIPTION)
-                .addPhoto(valueProvider.getFilePath("photos/image.png"));
+                .addPhoto(valueProvider.getFilePath("image.png"));
```

Comment:
???

Reply:
![image](https://user-images.githubusercontent.com/58187708/223843815-b8e64e97-c996-4011-9584-3b90f906b8b3.png)
we added directory for images

Reply:
![image](https://user-images.githubusercontent.com/58187708/223844248-2b6e4a55-d485-4934-9f79-f359ac35d459.png)
and I fixed that here

Reply:
I reverted the changes back